src/clients/Elsa.Api.Client/Resources/WorkflowInstances/Requests/ListWorkflowInstancesRequest.cs

```diff
@@ -13,64 +13,74 @@ public class ListWorkflowInstancesRequest
     /// Gets or sets the page number.
     /// </summary>
     public int? Page { get; set; }
-    
+
     /// <summary>
     /// Gets or sets the page size.
     /// </summary>
     public int? PageSize { get; set; }
-    
+
     /// <summary>
     /// Gets or sets the search term.
     /// </summary>
     public string? SearchTerm { get; set; }
-    
+
     /// <summary>
     /// Gets or sets the workflow definition id.
     /// </summary>
     public string? DefinitionId { get; set; }
-    
+
     /// <summary>
     /// Gets or sets the workflow definition ids.
     /// </summary>
-    [Query(CollectionFormat.Multi)] public ICollection<string>? DefinitionIds { get; set; }
-    
+    public ICollection<string>? DefinitionIds { get; set; }
+
     /// <summary>
     /// Gets or sets the correlation id.
     /// </summary>
     public string? CorrelationId { get; set; }
-    
+
     /// <summary>
     /// Gets or sets the version.
     /// </summary>
     public int? Version { get; set; }
-    
+
+    /// <summary>
+    /// Gets or sets whether the workflow instances have incidents or not.
+    /// </summary>
+    public bool? HasIncidents { get; set; }
```

Comment:
I'm not sur but we can have instances incidents just with status, failed or other ?

Reply:
It's a good question. Depending on the incident handling strategy applied to the workflow definition, it is possible that a workflow has incidents (faults), without the workflow itself being in the Faulted state.

There are currently two incident handling strategies:

- Fault: whenever an activity faults, the entire workflow faults. This is the default strategy.
- Continue with Incidents: keep the workflow in the Running state, even when an activity faults.

If we only had the first strategy, then you are correct and we would not need the "Has Incidents" filter.
But, since we also support the second strategy (and potentially others), the workflow status can remain to be Running, but with incidents; this is why we added this filter.